MVP_Clicker/Assets/Project/Scripts/Core/SaveSystemService/Model/ISaveSystemService.cs

```diff
@@ -0,0 +1,10 @@
+using Project.Scripts.Game.Base.GameData;
+
+namespace Project.Scripts.Game.Areas.SaveSystem
+{
+    public interface ISaveSystemService
+    {
+        public void SaveData(IGameData data);
+        public IGameData LoadData();
```

Comment:
This service have to be able to move to other games. So it should be more generic like `void SaveData<T>(T data);`